mmv1/templates/terraform/examples/forwarding_rule_externallb_byoipv6.tf.tmpl

```diff
@@ -0,0 +1,35 @@
+// Forwarding rule for External Network Load Balancing using Backend Services with IP Collection
+
+resource "google_compute_forwarding_rule" "{{$.PrimaryResourceId}}" {
+  provider              = google-beta
+  name                  = "{{index $.Vars "forwarding_rule_name"}}"
+  region                = "us-central1"
+  port_range            = 80
+  ip_protocol           = "TCP"
+  ip_version            = "IPV6"
+  load_balancing_scheme = "EXTERNAL"
+  ip_address            = "2600:1901:4457:1::/96"
+  network_tier          = "PREMIUM"
+  backend_service       = google_compute_region_backend_service.backend.id
+  ip_collection         = "https://www.googleapis.com/compute/beta/projects/tf-static-byoip/regions/us-central1/publicDelegatedPrefixes/tf-test-forwarding-rule-mode-pdp"
+}
+
+resource "google_compute_region_backend_service" "backend" {
+  provider              = google-beta
+  name                  = "{{index $.Vars "backend_name"}}"
+  region                = "us-central1"
+  load_balancing_scheme = "EXTERNAL"
+  health_checks         = [google_compute_region_health_check.hc.id]
+}
+
+resource "google_compute_region_health_check" "hc" {
+  provider           = google-beta
+  name               = "check-{{index $.Vars "backend_name"}}"
```

Comment:
If possible, this should just use the variable, so that it ends up with a `tf-test` prefix. (Or it could use a separate variable if you just want it to be more clear that it's a check for a backend.)

```suggestion
  name               = "{{index $.Vars "backend_name"}}"
```